Tests/AFNetworkActivityLoggerTests.m

```diff
@@ -265,5 +265,52 @@ - (void)testThatIndividualLoggerIsNotCalledWhenLoggerIsNilledOut {
     [manager invalidateSessionCancelingTasks:YES];
 }
 
+- (void)testThatResponseSerializerIsAFHTTPResponseSerializerAndResponseBodyIsText {
+    NSURL *baseURL = [NSURL URLWithString:@"https://httpbin.org"];
+    AFHTTPSessionManager *manager = [[AFHTTPSessionManager alloc] initWithBaseURL:baseURL];
+    manager.responseSerializer = [AFHTTPResponseSerializer serializer];
+    AFNetworkActivityTestLogger *testLogger = [AFNetworkActivityTestLogger new];
+    
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Finish Block Should Be Called"];
+    [testLogger setFinishBlock:^(NSURLSessionTask *task, id responseObject, NSTimeInterval elpasedTime, NSError *error) {
+        [expectation fulfill];
+    }];
+    [self.logger addLogger:testLogger];
+    [self.logger setLogLevel:AFLoggerLevelDebug];
+    [self.logger startLogging];
+    
+    [manager
+     POST:@"post"
```

Comment:
I find the code _if ([request HTTPBody]) {
        body = [[NSString alloc] initWithData:[request HTTPBody] encoding:NSUTF8StringEncoding];
    }_in AFNetworkActivityConsoleLogger.m is not test. so I find a POST request and has request body to cover it